master_write.py

```diff
@@ -27,69 +27,72 @@
 for rank in range(60, 61, 1):
     for roulette in range(20, 21, 1):
         for tournament in range(20, 21, 1):
-            if rank + roulette + tournament == population:
-                for reproduction in range(4, 5, 1):
-                    for crossover in range(92, 94, 2):
-                        for mutation_rate in range(25, 26, 1):
-                            for sigma in range(7, 8, 1):
-                                count = count+1
-                                color = iter(cm.rainbow(np.linspace(0, 1, 11)))
-                                temp_benchmark_gen = []
-                                temp_benchmark_gen2 = []
-                                run_name.append(f"{rank}_{roulette}_{tournament}_{reproduction}_{crossover}_{mutation_rate}_{sigma}")
-                                for test in range(1, 11):
-                                    temp_earliest = 50
-                                    temp_earliest2 = 50
-                                    c = next(color)
-                                    ave_fitness = []
-                                    max_fitness = []
-                                    ave_metric = []
-                                    min_metric = []
-                                    gen_num = []
-                                    for generation in range(0, 51, 1):
-                                        gen_num.append(generation)
-                                    for gens in range(0, 51):
-                                        runname=f"{rank}_{roulette}_{tournament}_{reproduction}_{crossover}_{mutation_rate}_{sigma}_{test}_{gens}"
-                                        fit = []
-                                        metric = []
-                                        with open(f"{runname}_generationData.csv") as F1:
-                                            txt_read = csv.reader(F1, delimiter=",")
-                                            for i, row in enumerate(txt_read):
-                                                if i > 3:
-                                                    fit.append(float(row[2]))
-                                                    metric.append(float(row[1]))
-                                        ave_fitness.append(mean(fit))
-                                        max_fitness.append(max(fit))
-                                        ave_metric.append(mean(metric))
-                                        min_metric.append(min(metric))
+            selections = rank + roulette + tournament
+            if selections == population:
+                for reproduction in range(0, 17, 4):
+                    for crossover in range(72, 101, 4):
+                        for mutation in range(4, 17, 4):
```

Comment:
These values don't match the ones in `estimate_time.py`.  Perhaps they should be passed from the command and/or from a config file (there's a thread that shows how to do it on Slack) instead of being hardcoded in both places.